ledger-core/vanilla/cachekit/core.go

```diff
@@ -121,7 +121,7 @@ func (p *Core) Add() (Index, GenNo) {
 // Delete can ONLY be called once per index, otherwise counting will be broken
 func (p *Core) Delete(idx Index) {
 	p.alloc.Dec(idx) // this will remove +1 for oldest tracking
```

Comment:
please, fix that commentary. they must be in a single block

Reply:
fixed